packages/compiler/core/types.ts

```diff
@@ -1,5 +1,6 @@
 import type { JSONSchemaType as AjvJSONSchemaType } from "ajv";
 import { Program } from "./program";
+import { JSONSchemaValidator } from "./schema-validator";
```

Comment:
```suggestion
import { JSONSchemaValidator } from "./schema-validator.js";
```

Reply:
No longer applicable as we define it in types.ts instead of importing it.